src/test/java/io/valkey/commands/jedis/SortedSetCommandsTest.java

```diff
@@ -1384,7 +1384,7 @@ public void zinterstore() {
   }
 
   @Test
-  public void zintertoreParams() {
```

Comment:
Spotting this required adding a pattern to ignore `z` prefixes and then recognizing that it was an interesting typo...